model/postgres/postgres_server.rb

```diff
@@ -87,7 +87,7 @@ def configure_hash
         configs[:recovery_target_time] = "'#{resource.restore_target}'"
       end
 
-      if standby? || doing_pitr?
+      if standby? || doing_pitr? || read_replica?
```

Comment:
We already enter this block for read replicas thanks to `doing_pitr?`, right? Do we need this?

Though, maybe `doing_pitr?` is not a good name for that method anymore. How about something like`restoring_from_blob_storage?`

Reply:
`doing_pitr?` is actually false for read-replica representative server because the timeline_access is `fetch`

Reply:
nah, I am wrong here, you're right